db/migrate/20150406022056_change_repayment_rate_to_integer_type_on_loan_requests.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeRepaymentRateToIntegerTypeOnLoanRequests < ActiveRecord::Migration
+  def change
+    change_column :loan_requests, :repayment_rate, 'integer USING CAST(repayment_rate AS integer)', default: 0
```

Comment:
Line is too long. [110/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.